### PR TITLE
Toast and Rating

### PR DESCRIPTION
**Pull request by vijaytembugade**
Toast and Ratings components.
Rating : https://vijaytembugade.github.io/XenonUI-Component-Library/Components/Rating/rating.html
Toast : https://vijaytembugade.github.io/XenonUI-Component-Library/Components/Toast/toast.html

### CODE REVIEW

File: Components/Cards/cards.css [hunk 1]
```diff
@@ -91,7 +94,6 @@
 .card-footer {
   border-top: 1px solid rgba(0, 0, 0, 0.1);
```
Comment (nshanbhag20001412) [inline Components/Cards/cards.css:95]: suggestion : you can use color variables instead of direct assigning

File: Components/Cards/cards.html [hunk 2]
```diff
@@ -15,6 +15,7 @@
     <link rel="stylesheet" href="../Badge/badge.css" />
     <link rel="stylesheet" href="../../default.css" />
     <link rel="stylesheet" href="./cards.css" />
+    <link rel="stylesheet" href="../Rating/rating.css" />
   </head>
   <body class="flex-container">
     <div style="width: 100%; margin: 3rem;">
```
Comment (nshanbhag20001412) [inline Components/Cards/cards.html:21]: avoid inline styling

File: Components/Rating/rating.css [hunk 3]
```diff
@@ -2,5 +2,5 @@
 
 }
 .rated{
-    color: #f8ea22;
+    color: #ffbb28;
```
Comment (nshanbhag20001412) [inline Components/Rating/rating.css]: try to use color variable

**Review by horsemaker — APPROVED** [review]
look nice & works smooth Vijay! 👍

**Review by nshanbhag20001412 — APPROVED** [review]

File: Components/Cards/cards.html (hunk 2 above)
Comment (vijaytembugade) [reply Components/Cards/cards.html:21]: Oh. Thats temoporary styles. I will remove it, in documenattion.

File: Components/Rating/rating.css (hunk 3 above)
Comment (vijaytembugade) [reply Components/Rating/rating.css]: Thanks for pointing it out. I will change those.

**Pull request merged**